File: 1-two-sum/1-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        for x in range (len(nums)):
            for y in range (x+1,len(nums)):
                if nums[y] + nums[x] == target:
                    return[x,y]
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        

File: 1-two-sum/1-two-sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        int[] res=new int[2];
        for(int i=0;i<nums.Length-1;i++ ){
        for(int a=i+1;a<nums.Length;a++){
            if(nums[i]+nums[a]==target){
                res[0]=i;
                res[1]=a;
                return res;
            }
        }
            
        }
    return res;
    }
}